spec/requests/api/my_predictions_controller_spec.rb

```diff
@@ -52,7 +52,9 @@
         FactoryBot.create(:prediction, creator: user, created_at: 1.minute.from_now)
       end
 
-      specify do
+      before { user && prediction && another_prediction }
+
+      it do
```

Comment:
Personally I much prefer `specify do` over `it do` because even though they're both nonsensical the former seems less nonsensical than the latter. But no need to fix unless you agree.

Reply:
Changed [here](https://github.com/bellroy/predictionbook/pull/201/commits/217b8869883011db803871bb6c4a96e9f2a263c1).